packs/feats/archetype/loremaster/loremaster-dedication.json

```diff
@@ -35,10 +35,7 @@
                 "predicate": [
                     "feat:bardic-lore"
                 ],
-                "selector": [
-                    "bardic",
-                    "bardic-lore"
-                ],
+                "selector": "bardic-lore",
```

Comment:
I believe multiple permutations are supplied for different localizations?
For example, Esoteric Lore has:
```
"selector": [
	"esoteric",
	"esoteric-lore",
	"lore-esoteric"
]
```